test/Microsoft.AspNetCore.SignalR.Client.Tests/LongPollingTransportTests.cs

```diff
@@ -166,13 +160,19 @@ public class LongPollingTransportTests
                 var longPollingTransport = new LongPollingTransport(httpClient);
                 try
                 {
-                    var connectionToTransport = Channel.CreateUnbounded<SendMessage>();
-                    var transportToConnection = Channel.CreateUnbounded<byte[]>();
-                    var channelConnection = new ChannelConnection<SendMessage, byte[]>(connectionToTransport, transportToConnection);
-                    await longPollingTransport.StartAsync(new Uri("http://fakeuri.org"), channelConnection, TransferMode.Binary, connection: new TestConnection());
+                    var pair = DuplexPipe.CreateConnectionPair(PipeOptions.Default, PipeOptions.Default);
+                    await longPollingTransport.StartAsync(new Uri("http://fakeuri.org"), pair.Application, TransferMode.Binary, connection: new TestConnection());
 
                     var exception =
-                        await Assert.ThrowsAsync<HttpRequestException>(async () => await transportToConnection.Reader.Completion.OrTimeout());
+                        await Assert.ThrowsAsync<HttpRequestException>(async () =>
+                        {
+                            async Task ReadAsync()
+                            {
+                                await pair.Transport.Input.ReadAsync();
+                            }
+
+                            await ReadAsync().OrTimeout();
```

Comment:
Looks like overkill o.O

Reply:
This was just to get a `Task` from the pipe awaiter.

Reply:
I could use one of the helpers I used in the other tests.